app/src/main/java/ch/sdp/vibester/activity/DownloadActivity.kt

```diff
@@ -21,16 +21,23 @@ import ch.sdp.vibester.api.ItunesMusicApi
 import ch.sdp.vibester.model.Song
 import okhttp3.OkHttpClient
 import org.w3c.dom.Text
+import java.io.File
 import java.lang.IllegalArgumentException
 /*
```

Comment:
```suggestion
import java.lang.IllegalArgumentException

/**
```
Doc: ** + missing space